{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2017-8925",
      "ASSIGNER": "cve@mitre.org"
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-404"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "https://github.com/torvalds/linux/commit/30572418b445d85fcfe6c8fe84c947d2606767d8",
          "name": "https://github.com/torvalds/linux/commit/30572418b445d85fcfe6c8fe84c947d2606767d8",
          "refsource": "CONFIRM",
          "tags": [
            "Issue Tracking",
            "Patch",
            "Third Party Advisory"
          ]
        },
        {
          "url": "http://www.kernel.org/pub/linux/kernel/v4.x/ChangeLog-4.10.4",
          "name": "http://www.kernel.org/pub/linux/kernel/v4.x/ChangeLog-4.10.4",
          "refsource": "CONFIRM",
          "tags": [
            "Release Notes",
            "Vendor Advisory"
          ]
        },
        {
          "url": "http://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=30572418b445d85fcfe6c8fe84c947d2606767d8",
          "name": "http://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=30572418b445d85fcfe6c8fe84c947d2606767d8",
          "refsource": "CONFIRM",
          "tags": [
            "Issue Tracking",
            "Patch",
            "Third Party Advisory"
          ]
        },
        {
          "url": "http://www.securityfocus.com/bid/98462",
          "name": "98462",
          "refsource": "BID",
          "tags": [
            "Third Party Advisory",
            "VDB Entry"
          ]
        },
        {
          "url": "http://www.debian.org/security/2017/dsa-3886",
          "name": "DSA-3886",
          "refsource": "DEBIAN",
          "tags": [
            "Third Party Advisory"
          ]
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "The omninet_open function in drivers/usb/serial/omninet.c in the Linux kernel before 4.10.4 allows local users to cause a denial of service (tty exhaustion) by leveraging reference count mishandling."
        }
      ]
    }
  },
  "configurations": {
    "nodes": [
      {
        "operator": "OR",
        "cpe_match": [
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:o:linux:linux_kernel:*:*:*:*:*:*:*:*"
          }
        ]
      },
      {
        "operator": "OR",
        "cpe_match": [
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:o:debian:debian_linux:8.0:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:o:debian:debian_linux:9.0:*:*:*:*:*:*:*"
          }
        ]
      }
    ]
  },
  "impact": {
    "baseMetricV2": {
      "cvssV2": {
        "vectorString": "AV:L/AC:L/Au:N/C:N/I:N/A:P",
        "accessVector": "LOCAL",
        "accessComplexity": "LOW",
        "authentication": "NONE",
        "confidentialityImpact": "NONE",
        "integrityImpact": "NONE",
        "availabilityImpact": "PARTIAL",
        "baseScore": 2.1
      },
      "severity": "LOW",
      "exploitabilityScore": 3.9,
      "impactScore": 2.9,
      "obtainAllPrivilege": false,
      "obtainUserPrivilege": false,
      "obtainOtherPrivilege": false,
      "userInteractionRequired": false
    }
  },
  "publishedDate": "2017-05-12T21:29Z",
  "lastModifiedDate": "2019-10-03T00:03Z"
}